examples/js/local.js

```diff
@@ -35,7 +35,7 @@ document.addEventListener('DOMContentLoaded', uploadFiles);
 
 // display uploaded image
 const loadMap = (image) => {
-  map.whenReady(function() {
+  map.whenReady(function () {
```

Comment:
Hi @jenny-alexander , this is looking good. I noticed you added some spaces on this line, 58 and 73. Its not a big deal but we have the previous style across the other js files and maybe we should keep the same here. What do you think? 
Thanks for working on this

Reply:
Hi @cesswairimu, thank you for your feedback. I changed my code editor preferences to not format on save. Can you please check again that the changes are as requested?
Thanks.